package.json

```diff
@@ -69,10 +69,16 @@
 		"zh_TW": "繁體中文"
 	},
 	"dependencies": {
-		"@godaddy-wordpress/coblocks-icons": "1.6.0"
+		"@godaddy-wordpress/coblocks-icons": "1.6.0",
+		"@wp-g2/components": "^0.0.164"
```

Comment:
Yeah, it seems to resolve the dependency problem, but I wonder if we are fixing the symptom and not the cause...